apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -156,18 +162,20 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
         </Versions>
       </StickyHeader>
       <BuildVariantProvider>
-        {buildVariants.map((b) => (
+        {buildVariants.map((b, i) => (
           <BuildRow
             key={b.id}
             build={b}
             handlePinClick={handlePinBV(b.id)}
+            isFirstBuild={i === 0}
             lastActiveVersionId={lastActiveVersionId}
             pinned={pins.includes(b.id)}
             projectIdentifier={projectIdentifier}
             versions={versions}
           />
         ))}
       </BuildVariantProvider>
+      <OnboardingTutorial guideCueRef={guideCueRef} />
```

Comment:
Optional if you are interested in merging this: I think you could gate this and the button on the result of `spruceWaterfallEnabled` from `useAdminBetaFeatures`.